### ISSUE TITLE
private-service-connect missing DNS config for packages.cloud.google.com

### ISSUE THREAD

**Issue by majduk** [terraform-google-modules/terraform-google-network#469]
### TL;DR

The private-service-connect adds only private DNS zones for googleapis.com, gcr.io, and pkg.dev however it is missing some domains described in [[1]](https://cloud.google.com/vpc/docs/configure-private-google-access#config):
- private.googleapis.com
  - packages.cloud.google.com
  - source.developers.google.com

References:
1. https://cloud.google.com/vpc/docs/configure-private-google-access#config

### Expected behavior

The module should add allow configuration of all the domains, eg via extra configuration parameter.

While direct CloudDNS use is possible, it is making the overall configuration less readable and harder to maintain.

### Observed behavior

The resources created/managed by this module are:

- Private DNS zone to configure private.googleapis.com.
- Private DNS zone to configure gcr.io.
- Private DNS zone to configure pdk.dev.

### Terraform Configuration

```hcl
module "private_service_connect" {
  source                     = "../../modules/private-service-connect"
  project_id                 = var.project_id
  network_self_link          = module.simple_vpc.network_self_link
  private_service_connect_ip = "10.3.0.5"
  forwarding_rule_target     = "all-apis"
}
```

### Terraform Version

```sh
1.3.3
```

**Comment by mike-callahan** [comment]
Taking a look at this
